dev/gpctl/cmd/clusters-update.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/gitpod-io/gitpod/ws-manager-bridge/api"
+)
+
+// clustersUpdateCmd represents the clustersUpdateCmd command
+var clustersUpdateCmd = &cobra.Command{
+	Use:   "update [cluster name] [property] [value]",
+	Short: "Update a cluster",
+	Long:  "Updates the [property] (score, max_score, or crodoned) of the cluster [cluster name] with the new value [value].",
+	Args:  cobra.ExactArgs(3),
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		conn, client, err := getClustersClient(ctx)
+		if err != nil {
+			log.WithError(err).Fatal("cannot connect")
+		}
+		defer conn.Close()
+
+		name := args[0]
+
+		var request *api.UpdateRequest
+		switch args[1] {
+		case "score":
```

Comment:
Riskying that this becomes picky: Why hot have this like the `register` command: Take multiple flags, and send a request for each flag? E.g., allow `gpctl clusters update --score 50 --max_score 100` in one cmd. IMO that would be the best solution UX wise, even if we have to send two requests for that behind the scenes.

Reply:
This leads me to the question: Why does the request allow to set one property only?